src/Gir/GenerationOptions.cs

```diff
@@ -20,8 +20,8 @@ public sealed class GenerationOptions
 		#endregion
 
 		#region Generation toggles
-		readonly bool compat;
-		public bool GenerateDocumentation { get { return !compat; } }
+		public bool Compat { get; }
```

Comment:
Can we make this compat bool back into a field and expose different generation toggles? It makes more sense to say:
`if (opts.GenerateInterfacesWithIPrefix)` rather than `if (opts.Compat)`

Reply:
fixed